liquidhaskell.cabal

```diff
@@ -271,4 +274,6 @@ test-suite test
                      tasty >= 0.10,
                      tasty-hunit >= 0.9,
                      tasty-rerun >= 1.1,
-                     transformers
+                     transformers,
+                     Cabal >= 1.18
```

Comment:
Yes, but the executable is a separate build target from the library, so it doesn't directly depend on cabal. Because of this separation, the executable target should only depend on packages that it imports **directly**, but I'll take a look :)